src/js/tracing/reactnativetracing.ts

```diff
@@ -111,6 +111,17 @@ export const reactNativeTracingIntegration = (
   };
 
   const setup = (client: Client): void => {
+    addDefaultOpForSpanFrom(client);
+
+    instrumentOutgoingRequests({
+      traceFetch: finalOptions.traceFetch,
+      traceXHR: finalOptions.traceXHR,
+      shouldCreateSpanForRequest: finalOptions.shouldCreateSpanForRequest,
+      tracePropagationTargets: client.getOptions().tracePropagationTargets || DEFAULT_TRACE_PROPAGATION_TARGETS,
+    });
+  };
+
+  const afterAllSetup = (): void => {
```

Comment:
● ReactNativeTracing › Tracing Instrumentation › With routing instrumentation › Cancels route transaction when app goes to background 

Seems to be failing on this specific PR, could you double-check if it broke with this changes?